### PR TITLE
Add Riot

### PR DESCRIPTION
**Pull request by aurri**
Thanks.

### CODE REVIEW

**Comment by aurri** [comment]
Fixed author field and squashed into a single commit.

File: files/riot/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "riotjs",
+  "repo": "muut/riotjs",
+  "files": {
+    "include": ["riot.js", "riot.min.js"]
```
Comment (jimaek) [inline files/riot/update.json]: I dont see these files here https://github.com/muut/riotjs/tree/v2.0.1
The bot will fail to update them in the future

**Comment by jimaek** [comment]
Also I think its safe to remove the unminified file. In production only the min is needed.

File: files/riot/update.json (hunk 1 above)
Comment (aurri) [inline files/riot/update.json]: Yea, these files weren't there in previous versions. They're already in `master` however, and will be included in all upcoming releases from now on.

**Comment by aurri** [comment]
Ideally, I would love to keep the unminified file as it can provide better feedback during the development (e.g. more meaningful errors) than the minified one. But if that's a requirement, no problem at all.

Let me know.

Thanks!

**Comment by jimaek** [comment]
Looks good then, thank you!

**Comment by aurri** [comment]
Awesome, thank you!

**Pull request merged**